src/System.IO.FileSystem/tests/Enumeration/RootTests.netcoreapp.cs

```diff
@@ -45,7 +47,8 @@ public void CanRecurseFromRoot()
                     }
 
                     // Should not get back a full path without a single separator (C:\foo.txt or /foo.txt)
```

Comment:
Why was it breaking, because there was only one file in the root directory on the CI machines, but not the official build machines?

Reply:
Order of what was in the root directory.

Reply:
I guess I'm not clear what was causing recursed.Current to have more than one element (cuasing the error in the bug). Since 
```
        public TResult Current => _current;

        object IEnumerator.Current => Current;
```
I read this as only returning one item?

Reply:
It was indexing the chars in the current path, not paths. :)